less/_defaults/_spacing-config.less

```diff
@@ -7,7 +7,7 @@
 @item-margin: 0.5rem;
 @item-padding: 1rem;
 
-@item-border-radius: 5px;
+@item-border-radius: 0.375rem;
```

Comment:
Can this be reverted please

Reply:
Of course. Can I why this is in px, just not clear to me.

Reply:
Oversight. You're correct in identifying that the value will need converting to rem to ensure relative scaling works with user defined font sizes.

User font medium, border radius 8px
<img width="89" alt="Screenshot 2023-05-22 at 10 28 50" src="https://github.com/adaptlearning/adapt-contrib-vanilla/assets/11569678/de972c2c-33fd-4f95-826b-6eea14d1bf53">

User font medium, border radius .5rem
<img width="85" alt="Screenshot 2023-05-22 at 10 28 59" src="https://github.com/adaptlearning/adapt-contrib-vanilla/assets/11569678/e90752f3-c358-4570-b7cb-700b95bfe485">

Both of the above screenshots look identical as is expected.

User font very large, border radius 8px
<img width="129" alt="Screenshot 2023-05-22 at 10 29 12" src="https://github.com/adaptlearning/adapt-contrib-vanilla/assets/11569678/e542f6c2-2a36-4efe-89dd-f2b19a273e42">

User font very large, border radius .5rem
<img width="136" alt="Screenshot 2023-05-22 at 10 29 17" src="https://github.com/adaptlearning/adapt-contrib-vanilla/assets/11569678/d88400a1-ab64-4e61-8318-9b809b82b72a">

The above two screenshots show a difference in border radius implementation when the user changes their browser font from medium (default) to very large. 

The reason for the revert is that the change sits outside of the remit of this pull request and should be it's own issue where it can be explained.

Tested in Chrome Mac OSX